docs/_templates/quickstart.rst

```diff
@@ -77,8 +77,25 @@ Carry out the steps in this order to ensure a successful analysis.
 
      Then restart the analysis with ``{{ start_analysis }}``.
      Additionally ``{{ start_analysis }}`` accepts modification to these
-     dictionaries when it is called. 
+     dictionaries when it is called see: <REPLACE WITH LINK TO START ANALYSIS DOCS>. 
      FIXME.
+     
+     For example:
+     
+     .. code-block:: python
+        pdf_kwargs.update(qmax=30)
+        mask_setting.update({'setting': 'first'})
+        mask_kwargs['alpha'] = 2
+    
+     Would change the PDF calculation Q max to 30 inverse Angstrom, reuse the 
+     mask from the first image of the scan on the other images, and change
+     the auto masking tolarance to 2 standard deviations.
+
+     .. DANGER::
+        Never run ``pdf_kwargs = {'qmax': 30}`` or the like with any of the
+        configuration dictionaries, this will detach them from the pipeline
+        and you will need to reset the analysis completely (exit ipython 
```

Comment:
for example, this is a good use of an example.  you said "you will have to reset the analysis", I think, Oh, what does that mean?  How do I do that? and immediately after it says "(exit ipython and run `setup_analysis` again)", so I think, ah, OK, I understand.

This is quickstart, so if the instructions are too long, put them somewhere else, but link to them.

But quick summary.  Nothing to change here....this is good....but we need something like this for the "run with configuration dictionaries" thing above.